jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -25,8 +26,22 @@
 from jax import test_util as jtu
 from jax import numpy as jnp
 
-class JaxToTfTestCase(jtu.JaxTestCase):
+import os
+
+# JAX2TF_CATEGORIZE_OUT is either not set, or contains the name of the
+# target file for the output.
+output_categories = os.getenv('JAX2TF_CATEGORIZE_OUT')
```

Comment:
Wouldn't it be easier to use if the file name is hard-coded and the path is computed relative to the current file?